site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs

```diff
@@ -4,11 +4,88 @@
+				<li>Patch - Feeds - Fixed condition when limit is greater than the number of entries</li>
+				<li>Patch - Toggle - Ajusted <code>&lt;detail&gt;</code> toggle behaviour to better match natural cycle when there is multiple details to toggle at once</li>
+			</ul>
+		</li>
+		<li>Maintenance
+			<ul>
+				<li>Patch - Fixed blocking condition in the build script related to sprites</li>
+			</ul>
+		</li>
+	</ul>
+	<p>Canada.ca (GCWeb)</p>
+	<ul>
+		<li>Template
+			<ul>
+				<li>Minor - Search template - New template for pages with search results.</li>
+				<li>Minor - Navigation element - A <code>&lt;nav&gt;</code> element was added for the skip nav link.</li>
+				<li>Minor - Font Awesome - A reference to font-awesome 5.8.1 was added inside the <code>&lt;head&gt;</code> tag.</li>
+				<li>Minor - RDFa + schema.org improvement to Canada wordmark - The RDFa markup has been modified, new property added, to improve SEO and how search engine interprete some information included in our template.</li>
+				<li>Patch - RDFa markup for main element - Fixed RDFa markup issue on the main element by adding <code>typeof="WebPageElement"</code> attribute to the <code>&lt;main&gt;</code> element.</li>
+			</ul>
+		</li>
+		<li>Styles and plugins
+			<ul>
+				<li>Minor - Custom CSS variant - New concept that allows integration of custom CSS for implementation variant of GCWeb by writing styles inside	new SASS files that are included in the build for pick-up.</li>
+				<li>Minor - Background colour -	Set a dark background to an element (e.g. well) with the class <code>.bg-dark</code> or a completely black one with <code>.bg-darker</code>.</li>
+				<li>Minor - Text colour- Set text in white with a CSS class called <code>text-white</code>. Works jointly with dark backgrounds.</li>
+				<li>Minor - Redacted text - Change of redacted text character to a full block when <code>.redacted</code> is in use.</li>
+				<li>Minor - Well - Provide a responsive well that takes the full width of its parent by adding the <code>.header-rwd</code> class to it.</li>
+				<li>Minor - URL manager and geomap - Added the possibility to prefill geomap configurations with action management.</li>
+				<li>Patch - Footer background image - Canadian flag was added on the top of the parliament on the landscpae image.</li>
+				<li>Patch - Carousel - Link underline override was removed on carousel.</li>
+				<li>Patch - Multimedia player - Top margin override was removed on multimedia player.</li>
+				<li>Patch - Menu - Fix menu navigation on different browsers when using a screen reader by running rigorous operability tests with JAWS and NVDA.</li>
+			</ul>
+		</li>
+		<li>Experimental
+			<ul>
+				<li>Minor - Bootstrap 4 spacers - CSS classes added for padding and margin spacing, e.g. <code>.p-0</code> for no padding and <code>.mb-sm-5</code> for extra-large bottom margin for small devices and up.</li>
```

Comment:
Remove "minor" but add a link to the appropriate design decision. Like "Voir les règles qui régissent les fonctionnalités expérimentales"